tests/Integration/NotifierTest.php

```diff
@@ -105,23 +106,49 @@ public function test_sendAlertsPerEmailToRecipient()
 
     public function test_sendAlertsPerEmailToRecipient_shouldUseDifferentSubjectDependingOnPeriod()
     {
-        $this->assertDateInSubject('week', 'week December 21 - 27, 2009');
-        $this->assertDateInSubject('day', 'Thursday, December 31, 2009');
-        $this->assertDateInSubject('month', 'December 2009');
+        $this->assertDateInSubject('week', $this->getMockMail('week')->getSubject());
```

Comment:
@adaqus Yes, this is why.

@mattab @tsteur What i want to achieve is that plugin have green tests for as many Piwik versions as possible or at least 2.15 and 2.16. The differences were only in tests because of differences in email subject sanitization in 2.16. If this is Notifier integration test and final subject depends on how Mail object works, we should obtain expected subject from Mail object. 

Now as I think of it, this is more a subject for a unit test, and there should dedicated method to generate a subject.

And then this test would be not required, because its name says it is more about generating the subject than testing actual mail shipment.